### PR TITLE
Added Orion HA recipe

### PR DESCRIPTION
**Pull request by Cerfoglg**

### CODE REVIEW

File: orion/1.12/mastermind.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: orion
+version: 1.12
+description: MongoDB Replicaset to work with Fiware IoT Services
```
Comment (taliaga) [inline orion/1.12/mastermind.yml]: description does not match name

File: orion/1.12/mastermind.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+protocol_type: HTTP
+ngsi_version: 1
+environment_variables:
+  - variable: ORION_LOG_LEVEL
+    name: Orion Log Level
+    description: The Logging level for Orion (e.g. DEBUG)
+    required: true
```
Comment (taliaga) [inline orion/1.12/mastermind.yml]: should not be required

File: orion/1.12/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  orion:
+    image: fiware/orion:1.12.0
+    ports:
+      - "1026:1026"
+    command: -logLevel ${ORION_LOG_LEVEL} -dbhost ${MONGO_SERVICE_URI} -rplSet ${REPLICASET_NAME} -dbTimeout 10000
```
Comment (taliaga) [inline orion/1.12/docker-compose.yml]: missing default values of env variables.

File: orion/1.12/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  orion:
+    image: fiware/orion:1.12.0
```
Comment (taliaga) [inline orion/1.12/docker-compose.yml]: let's take this chance to make this customizable

File: orion/1.12/mastermind.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  - variable: DOCKER_MTU
+    name: Docker MTU
+    description: The Docker MTU to use
+    required: true
+    managed: true
+    default: 1400
+services:
+  - service_type: mongo
+    name: Mongo Db Host
+    description: The host of the Mongo DB Replicaset to connect to
```
Comment (taliaga) [inline orion/1.12/mastermind.yml]: let's remove "replicaset" from here.

**Review by taliaga — CHANGES_REQUESTED** [review]

File: orion/1.12/mastermind.yml (hunk 1 above)
Comment (Cerfoglg) [reply orion/1.12/mastermind.yml]: @taliaga Oups. I was using another file as template, forgot to change this

File: orion/1.12/docker-compose.yml (hunk 3 above)
Comment (Cerfoglg) [reply orion/1.12/docker-compose.yml]: @taliaga The Service Manager hasn't had the little parsing problem fixed yet, so keeping them off for now. Will put them all back in once that's solved

**Review by taliaga — APPROVED** [review]

**Pull request merged**